help.go

```diff
@@ -152,24 +154,91 @@ func ShowAppHelp(c *Context) (err error) {
 	return nil
 }
 
+var shortFlagRegex = regexp.MustCompile(`^-`)
+
 // DefaultAppComplete prints the list of subcommands as the default app completion method
 func DefaultAppComplete(c *Context) {
-	for _, command := range c.App.Commands {
+	DefaultCompleteWithFlags(nil)(c)
```

Comment:
`(nil)(c)` is a bit of a confusing invocation, but I don't have enough context to suggest any alternatives here 🤔

Reply:
Yah kinda confusing. This method generates sub commands or flags using current/last command in argv. Problem is app it self is command so it doesn’t have any thing to pass